week1/homework/src/server.js

```diff
@@ -10,6 +10,33 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    const giveResult = (statusCode, object) => {
+      response.writeHead(statusCode, { 'Content-Type': 'application/json' });
+      response.write(JSON.stringify(object));
+    };
+
+    switch (request.url) {
```

Comment:
As your comment says,
> In this current version I used switch. It doesn't cause any ESLint error.

That's right! looking back to [google's style guide], it's better to use in this manner, I would say it's even better to use it when ever you have a code similar to this kind of thing **(not in the guides)**, when you have a variable definition and it's value one of a set of a values, I will go for the switch statement.

And that's why `Eslint` was complaining 🙂 

so you don't have to write always:
```javascript
if (variable === '/state') {
} else if (variable === '/other-state') {
} else {
}
```

in a switch it will be as the following which make it a lot easier to read:
```javascript
switch (variable) {
  case '/state':
  case '/other-state':
  default :
}
```

[google's style guide]: https://google.github.io/styleguide/jsguide.html#features-switch-statements

Reply:
I have never heard about: google's style guide! 
Thanks, it is very useful. The layout and how the information  are displayed , make it very  comfortable and clear.  I like it.